### PR TITLE
imgrate nacos-sdk-go to v2

### PR DESCRIPTION
**Pull request by PolarishT**
#### What type of PR is this?

feat: A new feature support nacos-sdk-go v2

#### What this PR does / why we need it (English/Chinese):
current hertz discovery do not support nacos-sdk-go-v2

#### Which issue(s) this PR fixes:
#96

### CODE REVIEW

**Comment by li-jin-gou** [comment]
可以参考 https://github.com/kitex-contrib/registry-nacos/blob/main/v2/README.md ，简略一点就可以，因为 API 是一样的。

**Review by li-jin-gou — CHANGES_REQUESTED** [review]
单测没过，需要修改

File: nacos/v2/go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module github.com/hertz-contrib/registry/nacosv2
```
Comment (Skyenought) [inline nacos/v2/go.mod:1]: module github.com/hertz-contrib/registry/nacos/v2

File: nacos/v2/exmaple/custom_config/client/main.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 CloudWeGo Authors.
```
Comment (Skyenought) [inline nacos/v2/exmaple/custom_config/client/main.go:1]: typo exmaple -> example

**Pull request closed without merging**